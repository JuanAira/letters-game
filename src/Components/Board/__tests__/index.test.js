import React from 'react';
import { render } from '@testing-library/react';
import { Board } from '../index';
import SharedProps from '../../../context/SharedProps';

describe('<Board /> component', () => {
  const setup = () => {
    const sharedProps = {
      keys: [1, 2, 3],
      nextPossibleValues: [],
    };

    const board = ['A', 'B', 'C'];
    const component = render(
      <SharedProps.Provider value={sharedProps} >
        <Board
        board={board}
        />
      </SharedProps.Provider>
    );

    return component;
  };

  it('should match snapshot - base', () => {
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });
});
