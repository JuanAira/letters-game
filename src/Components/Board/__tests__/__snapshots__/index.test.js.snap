// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Board /> component should match snapshot - base 1`] = `
<div>
  <div
    class="section-board"
  >
    <h3
      class="letter 0 letter--disabled"
    >
      A
    </h3>
    <h3
      class="letter 1 letter--select"
    >
      B
    </h3>
    <h3
      class="letter 2 letter--select"
    >
      C
    </h3>
  </div>
</div>
`;
